.travis.yml

```
@@ -35,20 +31,29 @@ before_install:
   - conda update --yes conda
 
 install:
+  # gjslint only runs in py27
+  - conda create --yes -n gjslint python=2.7 pip six
+  - source activate gjslint
+  - pip install https://github.com/google/closure-linter/archive/master.zip
+  - conda deactivate
+
   # install requests using conda to avoid a distutils error in certifi
   - conda create --yes -n travis python=$TRAVIS_PYTHON_VERSION pip numpy${NUMPY_VERSION} 'scipy>=0.17.0' matplotlib pandas flake8 pep8 jupyter coverage cython scikit-learn requests
   - source activate travis
   - conda install -c conda-forge phantomjs --yes
-  - pip install https://github.com/google/closure-linter/archive/master.zip 'sphinx<1.6' sphinx-bootstrap-theme coveralls
+  - pip install 'sphinx<1.6' sphinx-bootstrap-theme coveralls
   - pip install -e '.[all]' --verbose
   - npm install -g jsdoc
 
 script:
   - flake8 emperor/*.py tests/*.py scripts/*.py setup.py
-  # we can only run gjslint in a python 2.7.x environment
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then gjslint --custom_jsdoc_tags 'module,function,constructs,alias,default' 'emperor/support_files/js/*.js' 'tests/javascript_tests/*.js'; fi
   # we just check coverage in the latest version of NumPy
-  - coverage run tests/all_tests.py; coverage report
+  - coverage run tests/all_tests.py && coverage report
   - make -C doc html
+
+  # we can only run gjslint using py27
+  - conda deactivate
+  - conda activate gjslint
+  - gjslint --custom_jsdoc_tags 'module,function,constructs,alias,default' 'emperor/support_files/js/*.js' 'tests/javascript_tests/*.js'
```

You can collapse the two lines above into one:

```suggestion
  - conda run -n gjslint gjslint --custom_jsdoc_tags 'module,function,constructs,alias,default' 'emperor/support_files/js/*.js' 'tests/javascript_tests/*.js'
```